deployment/terraform/modules/docker_host/main.tf

```diff
@@ -0,0 +1,57 @@
+variable "cidr_block" {
+  type = "string"
+  default = "10.0.0.0/16"
+}
+
+variable "security_group_id" {
+  type = "string"
+  default = "my_security_group_id"
+}
+
+variable "security_group_name" {
+  type = "string"
+  default = "allow_docker"
+}
+
+
+
+resource "aws_vpc" "main" {
+  cidr_block = "${var.cidr_block}"
+}
+
+resource "aws_security_group" "allow_docker" {
+  name        = "${var.security_group_name}"
+  description = "Allow connection to docker engine from within VPC"
+  vpc_id      = "${aws_vpc.main.id}"
+
+  ingress {
+    from_port   = 0
+    to_port     = 2376
+    protocol    = "tcp"
+    cidr_blocks = ["${var.cidr_block}"]
+  }
+
+  ingress {
+    from_port   = 0
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["${var.cidr_block}"]
+  }
+
+  egress {  # Implicit with AWS, but Terraform requires that it be explicit:
+    from_port = 0
+    to_port = 0
+    protocol = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags {
+    Name = "${var.security_group_name}"
+  }
+}
```

Comment:
VPC config should be in a separate module ("network" or "vpc") - will be used by all resources of the stack (see #2600).

Reply:
Will do.